tests/rt.sh

```diff
@@ -400,7 +400,8 @@ if [[ $SINGLE_NAME != '' ]]; then
 fi
 
 if [[ $MACHINE_ID = hera.* ]] || [[ $MACHINE_ID = orion.* ]] || [[ $MACHINE_ID = cheyenne.* ]] || [[ $MACHINE_ID = jet.* ]]; then
-  RTPWD=${RTPWD:-$DISKNM/NEMSfv3gfs/develop-20201022/${RT_COMPILER^^}}
+  #RTPWD=${RTPWD:-$DISKNM/NEMSfv3gfs/develop-20201022/${RT_COMPILER^^}}
+  RTPWD=${RTPWD:-$DISKNM/DATM-MOM6-CICE5/update-20201025/INTEL}
```

Comment:
Will this baseline directory change break weather and s2s test? Is it possible you can add additional datm test under NEMSfv3gfs?

Reply:
DATM tests have been added to NEMSfv3gfs.